template/macro/2015/layout.html

```diff
@@ -90,61 +83,105 @@
 {% set story = image[0] %}
 
 {% set picture = image[0]|largest_image %}
-
-
-<tr><td>
-    <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#faf9f8">
-        <tr><td>{{ heading(heading_text=heading_text, link=heading_link, bg_colour=heading_colour, colour=heading_text_colour) }}</td></tr>
-        <tr><td align="left"><a href="{{ story.webUrl }}" border="0"><img src="{{ picture.file }}" width="100%" alt="{{ picture.typeData.altText }}" border="0" /></a></td></tr>
-        {% if image_only == False %}
-        <tr>
-            <td align="left">
-                {{ story_blocks.story(story, True) }}
-            </td>
-        </tr>
-        {% endif %}
-    </table>
-</td></tr>
+    <tr><td>
+        <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#faf9f8">
+            <tr><td>{{ heading(heading_text=heading_text, link=heading_link, bg_colour=heading_colour, colour=heading_text_colour) }}</td></tr>
+            <tr><td align="left"><a href="{{ story.webUrl }}" border="0"><img src="{{ picture.file }}" width="100%" alt="{{ picture.typeData.altText }}" border="0" /></a></td></tr>
+            {% if image_only == False %}
```

Comment:
I had a look at the code. It looks like this one is the same case as the "Don't modify a variable inside the condition evaluation". We didn't do it and it might be appearing as a commit because we deleted it for a while and then reverted it back. Can be validated by comparing to template/macro/layout.html, once again.